src/notifications/setup_nds.md

```diff
@@ -0,0 +1,26 @@
+# Setup a Notification Delivery Service (NDS)
+
+Receiving push notifications involves using a NDS as an intermediary to receive the webhook event from one of the SDK services. These can be currently one of several services that provide information about events that the Breez SDK registers for. For example, payment events from the LSP or swap transaction confirmation events from the chain service. The NDS then processes this information and dispatches a push notification to the intended mobile device, ensuring the user receives timely updates about incoming events. This architecture necessitates vendors setup up and maintain their own NDS, tailored to handle and forward these notifications efficiently. An example payload when a `payment_received` POST request to the webhook URL contains the following JSON formatted structure:
+
+```
+{
+    "template": "payment_received",
+    "data": {  
+        "payment_hash": [payment hash]
+    }
+}
+```
+
+The vendor needs to run their own NDS because it is configured to send push notifications to your application users and therefore should be configured with the required keys and certificates. You can use our [reference NDS implementation](https://github.com/breez/notify) as an implementation starting point or as is. Our implementation of the NDS expects URLs in the following format:
```

Comment:
```suggestion
The vendor needs to run their own NDS because it is configured to send push notifications to your application users and therefore should be configured with the required keys and certificates. You can use our [reference NDS implementation](https://github.com/breez/notify) as a starting point or as is. Our implementation of the NDS expects URLs in the following format:
```